apps/linows/src/css/components/translate.css

```
@@ -1,14 +1,29 @@
 /* Translation panel - replaces the results row in the main-pane stack
  * (translate mode hides both result columns). */
 .translate-panel {
+    --translate-pad-y: 10px;
+    /* Bottom rows bleed out of the card padding onto the pane-footer gutter. */
+    --translate-bleed: calc(var(--row-inset) - var(--content-padding));
     flex: 1;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)